docs/rfc/rfc-018-bls-agg-exploration.md

```diff
@@ -414,12 +433,50 @@ initial audit by NCC group, a firm I'd never heard of.
 cryptography. It is not audited and is documented as 'as-is', although
 development appears to be active so formal verification may be forthcoming.
 
+#### CIRCL
+
+[CIRCL][circl] is a go-native implementation of several cryptographic primitives,
+bls12-381 among them. The library is written and maintained by Cloudflare and
+appears to receive frequent contributions. However, it lists itself as experimental
+and urges users to take caution before using it in production.
+
+### Light Client Change
+
+Implementing BLS signature aggregation in Tendermint would pose issues for the
+light client. The light client currently validates a subset of the signatures
+on a block when performing the verification algorithm. This is no longer possible
+with an aggregated signature. Aggregated signature verification is all-or-nothing.
```

Comment:
The function that checks this is also called form the [evidence reactor](https://github.com/tendermint/tendermint/blob/dd4fee88ef6e4e0e5cf8eab0bf5a3e078df355c9/internal/evidence/verify.go#L177) (and currently I added it into the blocksync verification, but we can discuss this).